indico/web/client/styles/base/_reset.scss

```diff
@@ -20,23 +20,29 @@ isindex {
 }
 
 fieldset,
-legend {
+legend,
+ul {
```

Comment:
I guess this is causing the problem with lists I mentioned in a comment on the PR itself?